src/libraries/System.Drawing.Primitives/src/System/Drawing/SystemColors.cs

```diff
@@ -47,5 +51,24 @@ public static class SystemColors
         public static Color Window => Color.FromKnownColor(KnownColor.Window);
         public static Color WindowFrame => Color.FromKnownColor(KnownColor.WindowFrame);
         public static Color WindowText => Color.FromKnownColor(KnownColor.WindowText);
+
+        /// <summary>
+        /// When <see langword="true"/>, system <see cref="KnownColor"/> values will return
+        /// the alternative color set (as returned by <see cref="SystemColors"/> statics or
+        /// <see cref="Color.FromKnownColor(KnownColor)"/>). This is currently "dark mode"
+        /// variants of the system colors.
+        /// </summary>
```

Comment:
Does setting this to true impact already constructed instances of Color's or any other types, or do those remain unchanged?

It seems folks would be free to toggle this while the app is running, so it would be good to describe how that impacts already created objects.

Reply:
I've called out more explicitly in the remarks about how Color behaves.

Reply:
Is that expected for users?  Does it raise the right events when it happens so that controls get updated consistently?

For example - when I change the value of this property and an application is running - will the application update all at once, or only when it happens to redraw controls for some reason?  Do we need to make the setter send some event (window message) or tell callers to do something when changing this?

Reply:
WinForms listens to WM_SYSCOLORCHANGE and refreshes everything. Accessibility readiness depends on this.

Reply:
Should this API send that message when the user changes the value of this bool?  Or do you want to document that applications should do so themselves?

Reply:
Well, it is a weird thing as this isn't a system-wide change. Broadcasting wouldn't make sense I think.

The expectation is that you would let your desktop runtime (that is Windows Forms) handle this for you generally speaking. I'll discuss this particular aspect with @KlausLoeffelmann and look at coming back with more documentation details as we finish implementing the upstream feature.